serverless/serverless
# Update Stack Policy Deny action

## What did you implement:

Updated the AWS serverless.yml documentation to add the `resources` key to the stackPolicy Deny action example, without which key, the Deny action appears to deny nothing.

Closes #nothing 

## How did you implement it:

## How can we verify it:

To verify, try a Deny action without a resources key, and then with a resources key and you should find that the latter denies action as desired.

## Todos:


***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
